prospector.yml

```diff
@@ -7,16 +7,39 @@ uses:
   - django
   - celery
 
+# TODO: review these paths and try to remove them from here
 ignore-paths:
   - docs/
   - settings/
   - common/
+  - constants.py
+  - urls.py
+  - wsgi.py
+  - acl
+  - builds
+  - cdn
+  - core
+  - doc_builder
+  - donate
+  - gold
+  - integrations
+  - locale
+  - notifications
+  - oauth
+  - payments
+  - profiles
+  - projects
+  - redirects
+  - api
+  - rtd_tests
+  - search
+  - settings
+  - templates
+  - vcs_support
```

Comment:
This will now ignore all warnings for all these modules, we should fix all the errors before merging the prospector files.

Reply:
Yes. I copied these ignores from our current configuration. We are already ignoring all these files.

Fixing the errors from all these files is outside the scope of this PR.

Reply:
OK. I remove them all. We will be fixing them as we go when touching/modifying those files.

Reply:
So, are going to see new warnings every time we touch existing code only? This is, not in the existing/unchanged code?

Reply:
Yes. Unfortunately, prospector does not work over "chunks of codes" as darker. So, it will run over the whole file.

Reply:
I'm fine fixing the errors while changing something else in those files, @ericholscher may have some thoughts here.

Reply:
We can blackify the whole code, but I don't think we can lint it. Unfortunately, some linting issues are not solved automatically by any tools 🤷🏼

Reply:
Ah, true. Good point.  👍